app/src/ui/banners/open-thank-you-card.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react'
+import { LinkButton } from '../lib/link-button'
+import { Octicon, OcticonSymbol } from '../octicons'
+import { Banner } from './banner'
+
+interface IOpenThankYouCardProps {
+  readonly onDismissed: () => void
+  readonly onOpenCard: () => void
+  readonly onThrewCardAway: () => void
```

Comment:
Ultra nit: should this be… `onThrowCardAway`? (present instead of past)
```suggestion
  readonly onThrowCardAway: () => void
```

Reply:
done